app/common/src/services/Backend.ts

```diff
@@ -850,6 +850,8 @@ export const COLORS = [
   { lightness: 22, chroma: 13, hue: 252 },
 ] as const satisfies LChColor[]
 
+export const FALLBACK_COLOR = COLORS[0]
+
```

Comment:
Why color-related code (above and below) is in `Backend.ts`? If I would need any helper for LCH, I would never think about looking here.

Reply:
these are a fixed set of colors to be used for labels specifically, not sure if it makes more sense to be in its own module